homeassistant/components/hausbus/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "domain": "hausbus",
+  "name": "Haus-Bus",
+  "codeowners": ["@Zero545"],
+  "config_flow": true,
+  "dependencies": [],
+  "documentation": "https://www.home-assistant.io/integrations/hausbus",
+  "homekit": {},
+  "iot_class": "local_push",
+  "loggers": ["pyhausbus"],
+  "requirements": ["pyhausbus==1.0.0"],
```

Comment:
It looks like this is not a generic requirement but written specifically for Home Assistant (it has a `homeassistant` package). Why is this necessary and what is Home Assistant specific about it?

Reply:
I also don't see the source on GitHub linked from the PyPI package. Is there?

Reply:
The pyhausbaus package is maintained by the owner of [Haus-Bus.de](https://github.com/hausbus). The source code repository is located [here](https://github.com/hausbus/homeassistant). I am not sure, why it is not shown on the pypi page. I did add the link to the repo in the [pyproject.toml](https://github.com/hausbus/homeassistant/blob/main/pyproject.toml). 

There is nothing home assistant specific in that repository. It was named homeassistant, because the python library to access the Haus-Bus devices was created specifically for home assistant. But it is also possible to use it standalone. The python library offers an object to connect and communicate with the Haus-Bus devices via UDP.

Reply:
pypi page has been updated with a 1.0.1 release of pyhausbus